.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
How about adding .idea to the .gitignore so we can ignore the whole .idea folder?